syntaxes/nestedtext.yaml

```
@@ -58,7 +58,7 @@ repository:
             name: keyword.other.nestedtext
   dictionary-item:
     patterns:
-      - match: "^ *+([^\\[|{][^:]*+)(:( |\n))([^\n]*+)"
+      - match: "^\s*+([^\\[|{][^:]*+)(:( |\n))([^\n]*+)"
```

Since this is a string enclosed in double quotes you need to escape the backslash character with another backslash character
```suggestion
      - match: "^\\s*+([^\\[|{][^:]*+)(:( |\n))([^\n]*+)"
```